src/main/scala/mesosphere/marathon/health/HealthCheckWorkerActor.scala

```diff
@@ -74,7 +75,7 @@ class HealthCheckWorkerActor extends Actor with ActorLogging {
     }
 
   def http(task: MarathonTask, check: HealthCheck, port: Int): Future[Option[HealthResult]] = {
-    val host = task.getHost
+    val host = MarathonTasks.hostAddress(task)
```

Comment:
The most natural way would be a call to task (task.hostAddress). Since this is a proto, this is not possible. We should either think about to not use the proto object directly or use pimp my library extension pattern.